### PR TITLE
Fix(logs): defaultProps was removed

### PR DESCRIPTION
**Pull request by arielsrodriguez**
## Summary

defaultProps was removed

## Task

- not

## Affected sections

- src/components/Avatar/Avatar.tsx
- src/components/Badge/Badge.tsx
- src/components/Buttons/ButtonGroup.tsx
- src/components/Card/Card.tsx
- src/components/Form/Input/InputFile.tsx
- src/components/Form/Input/PercentageInput.tsx
- src/components/Form/Input/WrapperInput.tsx
- src/components/Image/Image.tsx
- src/components/ImageIcon/ImageIcon.tsx
- src/components/Kbd/Kbd.tsx
- src/components/Layout/Overflow/Overflow.tsx
- src/components/Layout/Row/Row.tsx
- src/components/Modal/Modal.tsx
- src/components/Pagination/Pagination.tsx
- src/components/Phase/Phase.tsx
- src/components/ProgressBar/ProgressBar.tsx
- src/components/ProgressCircle/ProgressCircle.tsx
- src/components/Stepper/Stepper.tsx
- src/components/Tag/Tag.tsx
- src/components/Typography/Text.tsx

## How did you test this change?

* Mannualy with storybook 🎨
* All tests was passed ✅

### CODE REVIEW

**Comment by arielsrodriguez** [comment]
@genixxavier @andres-bedolla @Misael840 @juandd360 @JuanDominguez1 can you help me with your reviews?🙏🏻

File: src/components/Form/Input/InputFile.tsx [hunk 1]
```diff
@@ -27,43 +27,46 @@ function InputFile({
...
   /**
    * It is a text message that goes on top of the box
    */
-  label,
+  label = 'Drag & drop your files or',
   /**
    * is the message that works to call the Input and with this upload files from a text button
    */
-  labelAction,
+  labelAction = 'browse from your device',
```
Comment (tbor00) [inline src/components/Form/Input/InputFile.tsx:54]: Mmm i don't know why we have some default messages in english and others in spanish

Comment (arielsrodriguez) [reply src/components/Form/Input/InputFile.tsx:54]: I translate the messages into English.

**Review by jcsoftdev — APPROVED** [review]

File: src/components/Kbd/Kbd.tsx [hunk 2]
```diff
@@ -9,37 +9,33 @@ export interface KBDProps {
...
-    </p>
-  )
-})
+    return (
+      <p
+        className="text-info dark:text-gray-400"
+        ref={ref as LegacyRef<HTMLParagraphElement>}
+      >
+        {kbds.map((text, index) => (
+          <React.Fragment key={`${index}-text`}>
```
Comment (andres-bedolla) [inline src/components/Kbd/Kbd.tsx:27]: From what I see this was already there, but we must avoid using the index as keys in the maps.

**Review by Misael840 — APPROVED** [review]

**Pull request merged**